backend/api/go_http_client/run_client/run_service/run_service_client.go

```diff
@@ -242,7 +242,7 @@ func (a *Client) ReportRunMetrics(params *ReportRunMetricsParams, authInfo runti
 }
 
 /*
-RetryRun res initiate a failed or terminated run
+RetryRun res initiates a failed or terminated run
```

Comment:
res initiates -> re-initiates

Reply:
Same as above. And just for the purpose of verification, the text in .proto file is actually correct. https://github.com/kubeflow/pipelines/blob/c38f36cdc14aa4b05a47aa13fd1e89b515279b92/backend/api/run.proto#L133
We'll just ignore the auto-generated text by swagger tool here.